azure-pipelines.yml

```diff
@@ -0,0 +1,36 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
```

Comment:
hi @KanishkT123 , let's chat offline some, since my recollection, and from re-looking at some [azdo docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/pr?view=azure-pipelines), is `trigger` and `pr` are AzDO pipeline yaml keywords -- and even work with github-repos that use AzDO pipelines -- but am not sure work from within [github actions themselves](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpull_requestpull_request_targetbranchesbranches-ignore).

Reply:
Ok, something said during the sprint review 'enlightened' me to this yml is actually being run (and expected to continue to be run?) as an AzDO pipeline ...

Reply:
This is being run within AzDo, since component governance cannot run within Github actions. This is therefore an Azure-pipelines.yml file, yep!